openedx/core/lib/api_utils.py

```diff
@@ -0,0 +1,73 @@
+"""Helper functions to get data from APIs"""
+from __future__ import unicode_literals
+import logging
+
+from django.core.cache import cache
+from edx_rest_api_client.client import EdxRestApiClient
+
+from openedx.core.lib.token_utils import get_id_token
+
+
+log = logging.getLogger(__name__)
+
+
+def get_api_data(api_config, user, api_name, resource, querystring=None, use_cache=False, cache_key=None):
+    """Fetch the data from the API using provided API Configuration and
+    resource.
+
+    Arguments:
+        api_config: The configuration which will be user for requesting data.
+        user (User): The user to authenticate as when requesting data.
+        api_name: Name fo the api to be use for logging.
```

Comment:
Why do you need this if you're passing the configuration model into the function? Just use api_config.API_NAME.